### ISSUE TITLE
Add better error handling in remote configuration utilization

### ISSUE THREAD

**Issue by eugenepeh** [reposense/RepoSense#367]
Some of the errors were not handled when the remote configuration files were not properly configured.

This can cause the program to crash halfway when analyzing multiple repositories.

**Comment by damithc** [comment]
What was the error? 
```
{
  "authors":
  [
    {
      "githubId": "leeyjjoel",
      "authorNames": ["leeyjjoel", "Lee Yi Jie Joel"]
    },
    {
      "githubId": "lessthanfree",
      "authorNames": ["lessthanfree", "David Goh"]
    },
    {
      "githubId": "tomforge",
      "authorNames": ["tomforge", "Foo Guo Wei"]
    },
    {
      "githubId": "yujiatay",
      "authorNames": ["yujiatay", "Yu Jia Tay"]
    },
    {
      "githubId": "truongthanh2606",
      "authorNames": ["truongthanh2606", "Truong Thanh"]
    }
  ]
}
```

**Comment by eugenepeh** [comment]
The error is that the displayName was not filled, hence the json parse it as null. Which is why we got a null pointer exception.

**Comment by eugenepeh** [comment]
Fixed in #397
